ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -438,7 +438,7 @@ private TaskTracker addLoadFunctionTasks(Context loadContext, BootstrapEventsIte
       // texts using new DB name. Currently it refers to the source database name.
     }
 
-    CreateViewDesc desc = new CreateViewDesc(dbDotView, table.getAllCols(), null, table.getParameters(),
+    CreateViewDesc desc = new CreateViewDesc(dbDotView, table.getCols(), null, table.getParameters(),
```

Comment:
The first variant of method - createViewTask: Line:427 seems to be unused. Should we remove it altogether?